code/game/objects/items/weapons/power_cells.dm

```diff
@@ -95,7 +95,7 @@
 	name = "infinite-capacity power cell!"
 	icon_state = "icell"
 	origin_tech =  null
-	maxcharge = 30000
+	maxcharge = 1000000000 //Good luck charging this, buff it if it gets fully charged
```

Comment:
You could use defines for this instead, such as

`maxcharge = ARBITRARILY_LARGE_NUMBER * 100000`